onnxruntime/core/providers/qnn/qnn_execution_provider.cc

```diff
@@ -496,7 +492,7 @@ QNNExecutionProvider::GetCapability(const onnxruntime::GraphViewer& graph_viewer
       if (partition && partition->sub_graph) {
         nodes_in_partition = partition->sub_graph->nodes.size();
 
-        if (nodes_in_partition == 1) {
+        if (nodes_in_partition == 1 && !load_from_cached_context) {
```

Comment:
It looks like `load_from_cached_context` is always `false`.

Reply:
Yes, should use is_qnn_ctx_model instead.

---
In reply to: [1472020695](https://github.com/microsoft/onnxruntime/pull/18865#discussion_r1472020695) [](ancestors = 1472020695)